files/grest/rpc/01_cached_tables/active_stake_cache.sql

```diff
@@ -22,6 +19,14 @@ CREATE TABLE IF NOT EXISTS GREST.ACCOUNT_ACTIVE_STAKE_CACHE (
   PRIMARY KEY (STAKE_ADDRESS, POOL_ID, EPOCH_NO)
 );
 
+CREATE TABLE IF NOT EXISTS GREST.LAST_TWO_EPOCHS_ACCOUNT_ACTIVE_STAKE_CACHE (
```

Comment:
OK to rename it to something smaller? Essentially what we're getting here is go and set snapshots (artificially), maybe stake_snapshot_cache?